### PR TITLE
feat: metabase OAM application

### PR DESCRIPTION
**Pull request by NimishKashyap**
### Checklist

- [x] Application license is compatible
- [x] I have created a directory in the form of `apps/<application_name>/<version>`
- [x] I have added the corresponding [metadata](https://docs.napptive.com/catalog/application_metadata.html) file
- [x] I have added the corresponding README file
- [x] I have tested the application in my own namespace and works as expected
- [x] I have tested that the application can be uploaded to my catalog namespace

#### What does this PR do?
This PR adds Metabase application to Napptive Catalog.

#### Which as the minimum requirements to launch the application?
1. 1vCPU
2. 4GB RAM.
3. 3GB Storage

### CODE REVIEW

File: apps/metabase/0.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Metabase Analytics tool
+
+This OAM application hosts Metabase Analytics Tool on Napptive with minimal configuration and authentication.
+
+Resources Requirement: 
+- CPU: 1vCPU
+- RAM: 4GB
```
Comment (Carmendelope) [inline apps/metabase/0.1.0/README.md]: With this requirements te application fails,
`metabase` requires memory: "4Gi"
 `metabase-db` and `metabase-test-db` require memory: "512M" each one.
could you adjust this? I tried it with 3Gi to `metabase` and it worked, but I do not know the reason  whty you set 4Gi.

File: apps/metabase/0.1.0/metabase.application.yaml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        env:
+          - name: POSTGRES_INITDB_ARGS
+            value: --data-checksums
+          - name: POSTGRES_DB
+            value: metabasedb
+          - name: POSTGRES_USER
+            value: metabase
+          - name: POSTGRES_PASSWORD
+            value: metabase
+    - name: metabase-test-db
```
Comment (Carmendelope) [inline apps/metabase/0.1.0/metabase.application.yaml:65]: Why do we have two databases?

**Review by Carmendelope — COMMENTED** [review]
Some comments

File: apps/metabase/0.1.0/README.md (hunk 1 above)
Comment (NimishKashyap) [reply apps/metabase/0.1.0/README.md]: Okay I'll update it

File: apps/metabase/0.1.0/metabase.application.yaml (hunk 2 above)
Comment (NimishKashyap) [reply apps/metabase/0.1.0/metabase.application.yaml:65]: We have two databases, one for metabase-related information, and the other for actually storing target analytics data as we don't want both to clash.

Comment (Carmendelope) [reply apps/metabase/0.1.0/metabase.application.yaml:65]: Screenshot 2023-11-21 at 10 31 54
So, what database should we configure here? Could you add it to the README file?

Comment (NimishKashyap) [reply apps/metabase/0.1.0/metabase.application.yaml:65]: The second one is marked as "metabase-test-db". Sure I'll add it to the Readme.

Comment (NimishKashyap) [reply apps/metabase/0.1.0/metabase.application.yaml:65]: Added

**Review by Carmendelope — APPROVED** [review]
Good!

**Pull request merged**